nucleus/common/common-util/src/main/java/com/sun/enterprise/loader/WatchProcessor.java

```diff
@@ -60,6 +60,7 @@
     protected boolean registered;
     protected boolean overflowed;
     protected boolean cancelled;
+    private int subscribers = 1;
```

Comment:
I take it that creating an instance is also considered subscribing to it. As this cannot get clear from the API alone I think a comment on the `subscribe` method is the best to point that out. WDYT?